.github/workflows/workflow.yml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Run unit tests
         run: make unit_test
 
+      - name: Run genesis tests
```

Comment:
Any reason on running genesis tests separately from integration/unit tests?

Reply:
No reason, I'm just experimenting with Github CI. For some reason, running it separately makes the whole thing pass. No idea why.

Reply:
Does not running it separately cause the test to be flaky then?

Reply:
If you run the tests of each package separately (i.e. `go test .`, `go test ./debounce`, etc.) instead of `go test ./...`, all of the tests will actually pass.